mrblib/itamae/resource_executor/user.rb

```diff
@@ -0,0 +1,82 @@
+module Itamae
+  module ResourceExecutor
+    class User < Base
+      def apply(current, desired)
+        if desired.exist
+          if exist?(desired.username)
```

Comment:
fixed in https://github.com/k0kubun/itamae-mruby/pull/3/commits/dccd0cb4b41dd92c3642806629909366712aaffa